README.md

```diff
@@ -0,0 +1,123 @@
+# Actix Basics
+
+Simple `actix_web` server example emulating the behavior of `https://jsonplaceholder.typicode.com/api/v1/users`
+
+## Running the Server
+
+```bash
+cargo run
+```
+
+This server, by default, binds and listens on `127.0.0.1:9090`.
+
+### Setting a binding with `BIND_URL` environment variable
+
+To override the default, set the environment variable `BIND_URL` to what you desire:
+
+```bash
+export BIND_URL="127.0.0.1:4242"
```

Comment:
Now that you have your app using an env var, upgrade to using the envy and dotenv crates.